documentation/staging/content/userguide/managing-operators/installation.md

```diff
@@ -245,6 +263,20 @@ Note that the Domain custom resource definition is shared.
 Do not delete the CRD if there are other operators in the same cluster
 or you have running domain resources.
 
+Beginning with Operator version 4.0, uninstalling an operator also removes the conversion webhook
+ deployment and its associated resources by default.
+Therefore, if you have multiple operators running, then, by default, an uninstall 
+of one operator will affect the other operators. The uninstall will not delete the conversion definition 
+in the domain CRD so you will be unable to create domains using `weblogic.oracle/v8` schema.
+If you want to prevent the uninstall of an operator  
+from having these side effects, then use one of the following two options:
+- [Install the conversion webhook]({{< relref "userguide/managing-operators/conversion-webhook#install-the-conversion-webhook" >}})
+ in a separate namespace using `webhookOnly=true` helm configuration value.
+- Use the `preserveWebhook=true` helm configuration value during operator installation with the `helm install` command.
+
+### Uninstall the WebLogic domain resource conversion webhook
+See [uninstall the conversion webhook]({{<relref "/userguide/managing-operators/conversion-webhook#uninstall-the-conversion-webhook" >}}) for the conversion webhook uninstallation details.
```

Comment:
Not sure this section is relay needed - maybe just add the See at the end of the previous section, and change "See' to "For more information".